src/Illuminate/Support/Collection.php

```diff
@@ -88,6 +88,32 @@ public function collapse()
     }
 
     /**
+     * Combine the collection with the given items
+     * using the collection as keys.
```

Comment:
this needs to all be on one line. If it won't fit, split it into short and long descriptions.